ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -126,7 +126,12 @@ public class RouterConfig {
       "router.update.op.metadata.reliance.timestamp.in.ms";
   public static final String ROUTER_UNAVAILABLE_DUE_TO_SUCCESS_COUNT_IS_NON_ZERO_FOR_DELETE =
       "router.unavailable.due.to.success.count.is.non.zero.for.delete";
-  public static final String ROUTER_REPAIR_WITH_REPLICATE_BLOB_ENABLED = "router.repair.with.replicate.blob.enabled";
+  // repair the blob with the on-demand replication on ttlupdate
+  public static final String ROUTER_REPAIR_WITH_REPLICATE_BLOB_ON_TTLUPDATE_ENABLED =
```

Comment:
We should rename the config to "router.repair.with.replicate.blob.on.ttlupdate.enabled".

Reply:
Currently ttlupdate on-demand replication is enabled. Changing it will make it disabled.

I'll use separated OSS and close source PR to change it at the same time.